sdk/bpf/c/inc/solana_sdk.h

```diff
@@ -526,6 +526,154 @@ uint64_t sol_try_find_program_address(
     uint8_t *bump_seed
 );
 
+/**
+ * BigNum sol_bignum_from_u32
+ *
+ * @param out_bn_ffi_ptr address of vector FfiBigNumber result structure (r/w)
+ * @param val_u32 unsigned 32 bit value to assign to the new object
+ */
+static uint64_t sol_bignum_from_u32(
+    const uint64_t *out_bn_ffi_ptr,
+    const uint64_t val_u32
+);
+/**
+ * BigNum sol_bignum_from_dec_str
+ *
+ * @param in_dec_str_ptr address of decimal string (r/o)
+ * @param in_size size of bytes array (r/o)
+ * @param out_bn_ffi_ptr address of vector FfiBigNumber result structure (r/w)
+ */
+static uint64_t sol_bignum_from_dec_str(
+    const uint64_t *in_dec_str_ptr,
+    const uint64_t in_size,
+    const uint64_t *out_bn_ffi_ptr
+);
+
+/**
+ * BigNum sol_bignum_add
+ *  Performs add and returns bignum for sum
+ * @param in_bn_lhs_ffi_ptr address of lhs input FfiBigNumber result structure (r/)
+ * @param in_bn_rhs_ffi_ptr address of rhs input FfiBigNumber result structure (r/)
+ * @param out_bn_ffi_ptr address of vector FfiBigNumber result structure (r/w)
+ */
+static uint64_t sol_bignum_add(
+  const uint64_t *in_bn_lhs_ffi_ptr,
```

Comment:
Same here and all following functions

Reply:
Added in 2289d94cbed61cf1250360cef0556a87e15c07cc